contrib/opencensus-ext-grpc/opencensus/ext/grpc/server_interceptor.py

```diff
@@ -107,28 +142,60 @@ def _start_server_span(self, servicer_context):
         )
 
         span.span_kind = span_module.SpanKind.SERVER
+
+        grpc_call_details = servicer_context._rpc_event.call_details
+        grpc_host = grpc_call_details.host.decode('utf-8')
+        grpc_method = grpc_call_details.method.decode('utf-8')
+
+        tracer.add_attribute_to_current_span(
+            COMPONENT, 'grpc'
+        )
+        tracer.add_attribute_to_current_span(
+            GRPC_METHOD, grpc_method
+        )
+
+        tracer.add_attribute_to_current_span(
+            HTTP_HOST, grpc_host
+        )
+        tracer.add_attribute_to_current_span(
+            HTTP_METHOD, 'POST'
```

Comment:
Sorry not too familiar with grpc, what's the different between `grpc_method` and `method`? Why is `method` always POST?

Reply:
The HTTP verbs for gRPC is restricted to POST as described in [gRPC specification](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#L25).

https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md#L25

The `grpc_method` is more like the name of the Function.

Here is comparison of gRPC  with HTTP API:
https://docs.microsoft.com/en-us/aspnet/core/grpc/comparison?view=aspnetcore-3.1